index.html

```diff
@@ -0,0 +1,206 @@
+        </p>
+      </article>
+    </section>
+
+    <!-- transformers -->
+    <section class="transformers">
+
+      <figure>
+        <img src="img/transformers.png">
+      </figure>
+
+      <article>
+        <header>
+          <h1>Transformers</h1>
+          <h3>Revenge of the Fallen</h3>
+        </header>
+        <section class="transformers-p">
+          <p><em>A noisy, underplotted, and overlong special effects extravaganza that lacks a human touch.</em></p>
+        </section>
+
+        <!-- rating -->
+        <section class="rating">
+          <h3 class="gray">Rotten Tomato</h3>
+          <h1 class="gray">Rating</h1>
+          <h1 class="progress">20%</h1>
+          <progress min="0" max="100" value="20">20%</progress>
+        </section>
+
+        <!-- date -->
+        <section class="date">
+          <h5 class="gray premiere">World Premiere</h5>
+          <h3>Den Haag Theatre</h3>
+          <h3 class="gray">The Netherlands</h3>
+          <time datetime="2011-06-23 pubdate">JUNE 23, 2011</time>
+        </section>
+      </article>
+
+    </section>
+
+    <!-- overflow -->
+    <section class="overflow">
+      <article class="overflow-article">
+        <header>
+        <figure class="water-pic">
+          <img src="img/water.png" alt="Water picture" title="Water picture">
+        </figure>
+        <div class="overflow-title">
+          <h1>Overflow</h1>
+        </div>
+        </header>
+        <p class="p1"><em>On the night of Aug 8, 1975, a line of people frantically piled sandbags atop Henan Province's <strong>Banqiao Dam</strong> while being battered by the worst storm ever recorded in the region. They were in a race with the rapidly rising Ru River to save the dam and the millions of people that lay sleeping downstream. It was a race theyw ere about to lose.</em></p>
+        <p class="p2"><em>Just after 1:00 am, the sky cleared and stars emerged from behind the storm clouds. There was an eerie calm as someone yelled, "The water level is going down! The flood is retreating!"</em></p>
+        <p class="p3"><em>There was little chance to enjoy that calm. One survivor recalled that a few seconds later it "sounded like the sky was collapsing and the earth was cracking." The equivalent of 280,000 Olympic-sized swimming pools burst through the crumbling dam, taking with it entire towns and as many as 171,000 lives.</em></p>
+      </article>
+    </section>
+
+    <!-- footer -->
+    <section>
+      <footer>
+        <p class="left"><strong>2013 CSSOff,</strong> Volume 3</p>
+        <!-- text align right on this -->
+        <p class="right"><strong>Route Gregory</strong>, Building 31 <br>
+          01630 Prevessin-Moens, France <br>
+          +41 22 767 61 11</p>
+      </footer>
+    </section>
+
+    <!-- form -->
+    <section class="form">
```

Comment:
Is this a `section` or a `fieldset`? 

http://www.anotheruiguy.com/ux-design-dev/_book/forms/fieldset.html